src/util/convertToVector3.js

```diff
@@ -8,12 +8,16 @@ import external from '../externalModules.js';
  * @param {Array|cornerstoneMath.Vector3} arrayOrVector3 Input array or Vector3
```

Comment:
Ohh, so `cornerstoneMath.Vector3` exists as a type? Is it defined anywhere for jsdoc to find it?

Reply:
Since we only pervasively use `cornerstoneMath.Vector2` and `cornerstoneMath.Vector3`, duplicating these definitions here isn't too bad I think. I think its better than the two alternatives: having it not linked to anything, or using a generic `Object` type.